### ISSUE TITLE
Trollolo crashes when trying to plot test board

### ISSUE THREAD

**Issue by promulo** [openSUSE/trollolo#25]
I have copied the [test board](https://trello.com/b/CRdddpdy/trollolo-testing-board) into my Trello account to try trollolo. However, when I try to plot using the test board, one of trollolo's Python scripts crashes.

**Steps to reproduce**
- clone trollolo repo and install requirements
```
$ ./bin/trollolo burndown-init --board-id=dtyEdYTh --output=test
$ cd test
$ ../bin/trollolo burndown
$ ../bin/trollolo plot 1
OR
$ ../bin/trollolo burndown --plot
```

**Expected**
Chart to be generated

**Actual**
```
(trollolo)[promulo@siren test]$ ../bin/trollolo burndown --plot
Traceback (most recent call last):
  File "/home/promulo/dev/trollolo/scripts/create_burndown.py", line 71, in <module>
    data = burndowndata.BurndownData(args)
  File "/home/promulo/dev/trollolo/scripts/burndowndata.py", line 15, in __init__
    self.setUnplannedTasksDayOne(burndown)
  File "/home/promulo/dev/trollolo/scripts/burndowndata.py", line 116, in setUnplannedTasksDayOne
    self.unplanned_tasks_day_one = burndown["days"][0]["unplanned_tasks"]["done"]
KeyError: 'done'
```

Using `.get("done", 0)` instead of `["done"]` [here](https://github.com/openSUSE/trollolo/blob/master/scripts/burndowndata.py#L116) avoids the issue but I'm not sure if the problem resides in the Python script, the YAML generation or the test board itself.

**Comment by cornelius** [comment]
@aosthof I think this is due to the calculation of unplanned tasks on day one. The bug only occurs, when there are unplanned tasks on day one. We usually don't have that, so we don't see this error. Here is an example for a file which creates the problem:

```yaml
---
meta:
  board_id: CRdddpdy
  sprint: 1
  total_days: 10
  weekend_lines:
  - 3.5
  - 8.5
days:
- date: '2016-03-17'
  updated_at: '2016-03-17T18:31:46+01:00'
  story_points:
    total: 16.0
    open: 13.0
  tasks:
    total: 13
    open: 9
  unplanned_story_points:
    total: 3.0
    open: 1.0
  unplanned_tasks:
    total: 2
    open: 1
```

We might want to add a test and fix it. Do you want to take this?

**Comment by aosthof** [comment]
Fixed by PR #30 now.
